### PR TITLE
Modification/landing page

### PR DESCRIPTION
**Pull request by philippgualdi**
Show more informations in the details section:
**DEMO**
Record-landing-page

**Versions**
Add a version list for marc21 record landing-page

landing_page_sidebar

### CODE REVIEW

File: invenio_records_marc21/templates/invenio_records_marc21/records/helpers/details.html [hunk 1]
```diff
@@ -6,14 +6,17 @@
...
 #}
 
-{%- from "invenio_records_marc21/records/macros/detail.html" import show_subtitles, show_detail, show_publisher %}
+{%- from "invenio_records_marc21/records/macros/detail.html" import show_subtitles, show_detail, show_publisher, show_dissertation, show_location_and_access, show_index_term_genre %}
 
 {% set details = marcrecord %}
 <h4 class="ui header">{{ _('Details')}}</h4>
 <hr class="thin-line">
-<div class="top-bottom-padded">  
+<dd class="top-bottom-padded" id="marc21-landing-page-details">
```
Comment (utnapischtim) [inline invenio_records_marc21/templates/invenio_records_marc21/records/helpers/details.html:14]: why do you change that to <dd></dd>?

Comment (philippgualdi) [reply invenio_records_marc21/templates/invenio_records_marc21/records/helpers/details.html:14]: I changed the tag name according to the RDM records landing page.

File: invenio_records_marc21/ui/records/wrappers.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#
+# Invenio-Records-Marc21 is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Records string wrapper."""
+
+from flask_babelex import gettext as _
+
+PERSONAL_CODES = {
```
Comment (utnapischtim) [inline invenio_records_marc21/ui/records/wrappers.py:15]: would it be possible to move that into a vocabulary? i would prefer not to hard code that dict

File: invenio_records_marc21/templates/invenio_records_marc21/records/helpers/details.html (hunk 1 above)
Comment (utnapischtim) [reply invenio_records_marc21/templates/invenio_records_marc21/records/helpers/details.html:14]: don't do that in that particularly thing. please leave it as it was.

File: invenio_records_marc21/ui/records/wrappers.py (hunk 2 above)
Comment (philippgualdi) [reply invenio_records_marc21/ui/records/wrappers.py:15]: Yes, but needs further investigation.

Comment (utnapischtim) [reply invenio_records_marc21/ui/records/wrappers.py:15]: ok, then we keep it that way for now and we open an issue to handle that later on

File: invenio_records_marc21/templates/invenio_records_marc21/records/macros/detail.html [hunk 3]
```diff
@@ -29,24 +30,23 @@
...
-    {% if key != "title" and key != "nonfiling_characters" %}
-      {% if key == "statement_of_responsibility" %}
-        {% set key = "Responsible" %}
-      {% elif key == "title_added_entry"%}
-        {% set key = "Added Title" %}
-      {% elif key == "title_added_entry"%}
-        {% set key = "Added Title" %}
-      {% elif key == "remainder_of_title"%}
-        {% set key = "Subtitle" %}
+    {% if key != "type_of_personal_name_entry_element" and key != "relator_code" %}
```
Comment (utnapischtim) [inline invenio_records_marc21/templates/invenio_records_marc21/records/macros/detail.html:46]: minor: would it not be easier to only allow explicitly the one allowed case?

File: invenio_records_marc21/translations/en/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -8,30 +8,78 @@ msgid ""
...
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.9.0\n"
+"Generated-By: Babel 2.9.1\n"
 
-#: invenio_records_marc21/services/schemas/access.py:55
-msgid "Embargo date must be in the future."
+#: invenio_records_marc21/errors.py:40
+msgid ""
+"The table you are looking for does not exist.\n"
```
[the remark this answers is not in the document]
Comment (philippgualdi) [reply invenio_records_marc21/translations/en/LC_MESSAGES/messages.po:24]: No this string is a string plotted on the CLI

File: invenio_records_marc21/ui/theme/assets/semantic-ui/less/invenio_records_marc21/landing-page.less [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  .row {
+    margin: 0.1rem 0.3rem 0.1rem;
+    padding: .1rem .1rem .1rem; // top left/right bottom
+  }
+  .column {
+    margin: 0rem;
+    padding: .4rem .9rem .4rem; // top left/right bottom
+  }
+
+}
```
[the remark this answers is not in the document]
Comment (philippgualdi) [reply invenio_records_marc21/ui/theme/assets/semantic-ui/less/invenio_records_marc21/landing-page.less]: There is a new line at the end of the file.

File: invenio_records_marc21/ui/theme/assets/semantic-ui/less/invenio_records_marc21/theme.less [hunk 6]
```diff
@@ -9,3 +9,4 @@
 
 @import "variables.less";
 @import "deposit.less";
+@import "landing-page.less";
```
[the remark this answers is not in the document]
Comment (philippgualdi) [reply invenio_records_marc21/ui/theme/assets/semantic-ui/less/invenio_records_marc21/theme.less]: There is a new line at the end of the file.

[the remark this answers is not in the document]
Comment (utnapischtim) [reply invenio_records_marc21/ui/theme/assets/semantic-ui/less/invenio_records_marc21/theme.less]: yes i know. know they are. i created those comments before we started using prettier

File: invenio_records_marc21/translations/en/LC_MESSAGES/messages.po (hunk 4 above)
[the remark this answers is not in the document]
Comment (utnapischtim) [reply invenio_records_marc21/translations/en/LC_MESSAGES/messages.po:24]: perfect. thx

**Review by utnapischtim — APPROVED** [review]

**Pull request merged**